drivers/bluetooth/btusb.c

```diff
@@ -4488,6 +4490,11 @@ static int btusb_probe(struct usb_interface *intf,
 	if (id->driver_info & BTUSB_MARVELL)
 		hdev->set_bdaddr = btusb_set_bdaddr_marvell;
 
+	if (id->driver_info & BTUSB_LOWER_LESCAN_INTERVAL) {
+		hdev->le_scan_interval = 0x0190;
+		hdev->le_scan_window = 0x000a;
+	}
```

Comment:
Maybe add a comment explaining those values, similar to the commit description (or you could just straight copy)? Without that they seem a bit magic-y.

Reply:
Agreed, added a small comment.

Reply:
Perfect, thanks!